src/i18n/en-US/index.ts

```diff
@@ -747,7 +747,7 @@ export default {
     dAppRewards: 'dApp rewards',
     done: 'Done',
     search: 'Search',
-    build: 'Build',
+    build: 'Build&Earn',
```

Comment:
```suggestion
    buildAndEarn: 'Build&Earn',
```

Reply:
@impelcrypto fixed in [30fc231](https://github.com/AstarNetwork/astar-apps/pull/1129/commits/30fc231b1dae22f4fc7ac1a2c09b96916d50f9cf)